src/gatk_gcnv_caller.sh

```
@@ -0,0 +1,655 @@
+    Calls GATK GermlineCNVCaller to call the actual CNVs.
+
+    This may run entirely in the parent job, or may be split to sub jobs by chromosome
+    dependent on if -iscatter_by_chromosome specified.
+
+    If splitting to sub jobs, _launch_sub_jobs is called, which holds the parent job until
+    all sub jobs have completed.
+    '''
+    mark-section "Running GermlineCNVCaller for the calculated basecounts using the generated ploidy file"
+    mkdir inputs/gCNV
+
+    if [ "$scatter_by_chromosome" == "true" ]; then
```

Shellcheck

[warning] scatter_by_chromosome is referenced but not assigned.

(SC2154)